components/style/binding_tools/regen.py

```diff
@@ -542,49 +541,11 @@ def zero_size_type(ty, flags):
         return 1
 
     print("OK")
+    print("(please test with ./mach test-stylo)")
```

Comment:
If you're running it inline you need the rest of style to compile, which might not be the case in the middle of development.

We could put these tests at the end of everything I guess (so that if it fails it doesn't cancel the next regen).